### PR TITLE
Dp 7191 hide sharebar on print

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/app/components/EditorialPage/EditorialPage.scss [hunk 1]
```diff
@@ -5,9 +5,13 @@
 
   &__body {
     background-color: $primary-light;
-    min-height: calc(100% - 278px);
+    min-height: calc(100vh - 330px);
```
Comment (chrisvanmook) [inline src/app/components/EditorialPage/EditorialPage.scss]: Where does the 330px come from? add a comment or create constant?

File: src/app/components/Dataset/Dataset.jsx [hunk 2]
```diff
@@ -40,7 +41,12 @@ const Dataset = ({
   })
 
   if (numberOfRecords === 0) {
-    return <NoResultsForSearchType message={<NoDatasetsMessage />} hideLoginLink />
+    return (
+      <>
+        <NoResultsForSearchType message={<NoDatasetsMessage />} hideLoginLink />
+        {!printMode && <ShareBar topSpacing={6} />}
```
Comment (chrisvanmook) [inline src/app/components/Dataset/Dataset.jsx]: ` {!printMode && <ShareBar topSpacing={6} />}` is used on 6 places in this PR. Is it possible to create a wrapper with this in it or something?

Comment (chrisvanmook) [reply src/app/components/Dataset/Dataset.jsx]: also see that the printMode is passed a lot of time to various components, so perhaps it's really worth it to create a HOC / wrapper of some sort if possible

File: src/app/components/EditorialPage/EditorialPage.scss (hunk 1 above)
Comment (aditudorache) [reply src/app/components/EditorialPage/EditorialPage.scss]: Thanks, comment added.

File: src/app/components/Dataset/Dataset.jsx (hunk 2 above)
Comment (aditudorache) [reply src/app/components/Dataset/Dataset.jsx]: Thanks! Refactored to ShareBar

**Pull request merged**